docs/maintenance/upgrade.md

```diff
@@ -33,6 +33,7 @@ To upgrade Codacy Self-hosted to the latest stable version:
 
     ```bash
     helm upgrade codacy codacy-stable/codacy \
```

Comment:
Maybe we should include an "helm repo update" command before this, in order to get the latest available versions